dogs-initial/src/main/java/com/lambdaschool/dogsinitial/exception/ResNotFoundException.java

```diff
@@ -0,0 +1,20 @@
+package com.lambdaschool.dogsinitial.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+public class ResNotFoundException extends RuntimeException
+{
+    private static final long serialVersionUID = 1L;
```

Comment:
Interesting that you choose a different name from convention. I would say though that error messages in general have full descriptive words for their class names anyway, so perhaps stick to `ResourceNotFoundException` instead next time (especially considering that "Res" can be an ambiguous contraction - there are other related words like "response" that start with it too.